ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -49,6 +58,7 @@ library
   import:           common-lib
   hs-source-dirs:   src/ouroboros-consensus
   exposed-modules:
+    Control.Concurrent.Class.MonadMVar.Strict.NoThunks
```

Comment:
I'm a bit unsure of adding this kind of "general" modules to Consensus, but I guess we are bridging NoThunks and strict-mvar here, so maybe there is not a better place

Reply:
I agree, but I'm also unsure about adding another `Util` module. We can keep it here for now, and move it somewhere better if we find a better place